udp.go

```diff
@@ -2,36 +2,27 @@ package session
 
 import (
 	"errors"
+	"log"
 	"net"
 	"strconv"
 	"time"
 
 	"github.com/nknorg/nkn-sdk-go"
-	"github.com/nknorg/nkngomobile"
 	"github.com/nknorg/tuna"
 )
 
-const (
-	sendUdpMetaDataRetries = 2 // retries to send udp meta data
-)
-
-func (c *TunaSessionClient) ListenUDP(addrsRe *nkngomobile.StringArray) (*UdpSession, error) {
-	acceptAddrs, err := getAcceptAddrs(addrsRe)
-	if err != nil {
-		return nil, err
-	}
-	if c.wallet == nil {
-		return nil, errors.New("ListenUDP wallet is empty")
+func (c *TunaSessionClient) ListenUDP() (*UdpSession, error) {
+	if len(c.listeners) == 0 {
+		log.Println("Please call Listen() to start TCP first")
```

Comment:
No need to print if you are returning the error here

Reply:
Yes, removed.